src/state/actions/dispatchArIOInteraction.ts

```
@@ -36,6 +36,8 @@
   signer,
   ao,
   antAo,
+  aoNetwork,
```

ESLint

[error] 'aoNetwork' is defined but never used.

(@typescript-eslint/no-unused-vars)